runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -407,7 +408,7 @@ private void updateLists()
 		updateList(layoutWhitelist, config.whitelistedLayouts());
 	}
 
-	private void updateList(ArrayList<String> list, String input)
+	private void updateList(Set<String> list, String input)
```

Comment:
Is there any reason the passed `list` should be a list or a set? This would be better accepting a `Collection` instead.